src/components/controls/mini-toolbar.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import Toolbar from '@material-ui/core/Toolbar'
+
+export class MiniToolbarUnconnected extends React.Component {
+  render() {
+    return (
+      <Toolbar
+        style={{ minHeight: '24px' }}
+        variant="dense"
+        disableGutters
+        classes={{ root: 'mini-toolbar' }}
+      >
+        {this.props.children}
+      </Toolbar>
+    )
+  }
+}
+
+export function mapStateToProps() {
+  return { }
+}
+
+const MiniToolbar = connect(mapStateToProps)(MiniToolbarUnconnected)
```

Comment:
this does not appear to need to be connected to the redux store at all, right? it's just a simple presentational component?